### PR TITLE
Updated README.md [9925]

### PR DESCRIPTION
**Pull request by rsanchez15**
This PR updates the README.md accordingly to the Discovery Server 2.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,189 @@
...
+* Windows
 
- Fast-RTPS v1.10 adoption of DDS API introduces xml and ABI incompatibilities with discovery server tests and examples
- with version number below v1.2.
 
- Fast RTPS dependencies as tinyxml must be accessible, either because Fast RTPS was build-installed defining
- THIRDPARTY=ON or because those libraries have been specifically installed.
+## Installation Guide
 
+In order to use the Discovery Server tool, its necessary to have a compatible version of
```
Comment (jparisu) [inline README.md:46]:
```suggestion
In order to use the Discovery Server tool, it is necessary to have a compatible version of
```

File: README.md [hunk 2]
```diff
@@ -76,178 +193,26 @@ In order to avoid using vcstool the following repositories should be downloaded
...
-
-+ The participant profile **builtin** tag contains a **discovery_config** tag where all discovery-related info is
- gathered. This new tag contains the following new elements:
-	- a **discoveryProtocol** tag, where the discovery type can be specified through the `DiscoveryProtocol_t`
-    enumeration.
-	- a **discoveryServersList** tag, where the server or servers linked with a participant can be specified.
-	- a **clientAnnouncementPeriod** tag, where the time span between PDP metatraffic exchange can be specified.
-
-Below we provide an example XML participant profile using these new *tags*:
 ```xml
```
Comment (EduPonz) [inline README.md:214]: comments in here look weird

File: README.md [hunk 3]
```diff
@@ -76,178 +193,26 @@ In order to avoid using vcstool the following repositories should be downloaded
...
 
-3. In order to run the tests use the following command:
-```bash
-[BUILD]$ colcon test --base-paths [SOURCES] --packages-select discovery-server
-```
-   note that only the test matching the build (step 2) configuration would run.
+-   The participant profile ``builtin`` tag contains a ``discovery_config`` tag where all discovery-related info is
+    gathered. This new tag contains the following new XML child elements:
+	- ``<discoveryProtocol>``: specifies the discovery type through the ``DiscoveryProtocol_t`` enumeration.
+	- ``<discoveryServersList>``: specifies the server or servers linked with a Client/Server.
```
Comment (EduPonz) [inline README.md:209]:
```suggestion
	- ``<discoveryServersList>``: specifies the server or servers to which a Client/Server connects.
```

File: README.md [hunk 4]
```diff
@@ -293,1024 +258,40 @@ Below we provide an example XML participant profile using these new *tags*:
...
-    creating a **ServerList** (only if this client references a single server). Based on this prefix the discover-server
-    parser would search for the corresponding server locators within the config file.
-	- **listening_port** specifies a physical port where to listen for incoming traffic. This attribute is mandatory in
-    TCP transport (client wouldn't receive other clients traffic without it). When using the TCPv4 the format is:
-    [XXX.XXX.XXX.XXX:]XXXX where the IP address is the client's WAN address that must be specified if we want the client
-    to be reachable from outside a local NAT.
-	- **creation_time** introduced for testing purposes specifies in seconds when a client must be created.
-	- **removal_time** introduced for testing purposes specifies in seconds when a client must be destroyed.
+The discovery server binary (named after the pattern ``discovery-server-X.X.X(d)`` where ``X.X.X`` is the version
+number and the optional *d* denotes a debug builds) is set up from one XML profiles files passed as command-line arguments.
```
Comment (EduPonz) [inline README.md:264]:
```suggestion
number and the optional *d* denotes a debug build) is set up from one XML profiles files passed as command-line arguments.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]
Also check italics/`code-block` consistency

File: README.md [hunk 5]
```diff
@@ -1,72 +1,190 @@
 # eProsima Discovery Server
 
-The [current standard](http://www.omg.org/spec/DDSI-RTPS/2.3) in section 8.5 specifies a non-centralized, distributed
-simple discovery mechanism for RTPS. This mechanism was devised to allow interoperability among independent  vendor-specific implementations but is not expected to be optimal in every environment. There are several scenarios were the simple discovery mechanism is unsuitable or plainly cannot be
+The [RTPS standard](http://www.omg.org/spec/DDSI-RTPS/2.3) specifies in section 8.5 a non-centralized, distributed
```
Comment (EduPonz) [inline README.md:3]:
```suggestion
The [RTPS standard](http://www.omg.org/spec/DDSI-RTPS/2.3) specifies in section 8.5 a non-centralized, distributed,
```

File: README.md [hunk 6]
```diff
@@ -1,72 +1,190 @@
...
+simple discovery mechanism. This mechanism was devised to allow interoperability among independent
```
Comment (EduPonz) [inline README.md:4]:
```suggestion
simple discovery mechanism. This mechanism was devised to allow interoperability between independent
```

File: README.md [hunk 7]
```diff
@@ -1,72 +1,190 @@
...
+vendor-specific implementations but it is not expected to be optimal in every environment.
+There are several scenarios were the simple discovery mechanism is unsuitable or plainly cannot be
```
Comment (EduPonz) [inline README.md:6]:
```suggestion
There are several scenarios where the simple discovery mechanism is unsuitable, or it just simply cannot be
```

File: README.md [hunk 8]
```diff
@@ -1,72 +1,190 @@
...
- with version number below v1.2.
 
- Fast RTPS dependencies as tinyxml must be accessible, either because Fast RTPS was build-installed defining
- THIRDPARTY=ON or because those libraries have been specifically installed.
+## Installation Guide
 
+In order to use the Discovery Server tool, it is necessary to have a compatible version of
+[eProsima Fast DDS](https://eprosima-fast-rtps.readthedocs.io/en/latest/) installed (over release 2.0.2).
+*Fast DDS* dependencies as tinyxml must be accessible, either because *Fast DDS* was build-installed defining
+THIRDPARTY=ON or because those libraries have been specifically installed.
```
Comment (EduPonz) [inline README.md:50]:
```suggestion
`THIRDPARTY=ON|FORCE` or because those libraries have been specifically installed.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Review by EduPonz — APPROVED** [review]
LGTM

**Pull request merged**